evaluator/evaluator/tasks/celery.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+from celery import Celery
+
+logger = logging.getLogger(__name__)
+
+# initialize env vars
```

Comment:
why not use the /app/settings classes here too?

Reply:
Hi, what do you mean? We took our orientation from the Celery scripts from you guys :D

like here: https://github.com/UKP-SQuARE/square-core/blob/master/square-model-inference-api/inference_server/tasks/celery.py#L1-L14 

```py


import logging
import os

from celery import Celery

logger = logging.getLogger(__name__)

# use env vars
rabbitmq_user = os.getenv("RABBITMQ_DEFAULT_USER", "ukp")
rabbitmq_password = os.getenv("RABBITMQ_DEFAULT_PASS", "secret")

redis_user = os.getenv("REDIS_USER", "ukp")
redis_password = os.getenv("REDIS_PASSWORD", "secret")
redis_host = os.getenv("REDIS_HOST", "redis")
```

Reply:
@timbmg updated :)